community/kernel/src/test/java/org/neo4j/kernel/impl/store/PropertyKeyTest.java

```diff
@@ -49,7 +50,7 @@ public void lazyLoadWithinWriteTransaction() throws Exception
         // Given
         FileSystemAbstraction fileSystem = fs.get();
         File dir = new File( "dir" ).getAbsoluteFile();
-        BatchInserter inserter = BatchInserters.inserter( dir, fileSystem );
+        BatchInserter inserter = BatchInserters.inserter( dir, new UncloseableDelegatingFileSystemAbstraction( fileSystem ) );
```

Comment:
With suggested change to BatchInserters and BatchInserterImpl we can get rid of this and let PropertyKeyTest take full responsibility for its fileSystem.